python_modules/dagster/dagster/_core/definitions/reconstruct.py

```diff
@@ -163,67 +163,72 @@ def get_python_origin_id(self) -> str:
         return self.get_python_origin().get_id()
 
     # Allow this to be hashed for use in `lru_cache`. This is needed because:
-    # - `ReconstructablePipeline` uses `lru_cache`
-    # - `ReconstructablePipeline` has a `ReconstructableRepository` attribute
+    # - `ReconstructableJob` uses `lru_cache`
+    # - `ReconstructableJob` has a `ReconstructableRepository` attribute
     # - `ReconstructableRepository` has `Sequence` attributes that are unhashable by default
     def __hash__(self) -> int:
         if not hasattr(self, "_hash"):
             self._hash = hash_collection(self)
         return self._hash
 
 
-@whitelist_for_serdes
-class ReconstructablePipeline(
+@whitelist_for_serdes(
+    storage_name="ReconstructableJob",
+    storage_field_names={
+        "job_name": "pipeline_name",
+    },
+)
+class ReconstructableJob(
```

Comment:
shouldn't the storage name here be "ReconstructablePipeline"? I guess maybe it's fine that it's reconstructable job, as it should only ever be used within user code, which should all be running the same version of dagster? @alangenfeld thoughts?

Reply:
You're right, that's a legit mistake